src/api/app/helpers/webui/request_helper.rb

```diff
@@ -7,13 +7,39 @@ module Webui::RequestHelper
     'superseded' => 'red'
   }.freeze
 
+  STATE_BOOTSTRAP_COLORS = {
+    'new' => 'info',
+    'review' => 'warning',
+    'accepted' => 'success',
+    'declined' => 'danger',
+    'revoked' => 'dark',
+    'superseded' => 'secondary'
+  }.freeze
+
+  STATE_BOOTSTRAP_ICONS = {
+    'new' => 'fa-code-branch',
+    'review' => 'fa-search',
+    'accepted' => 'fa-check',
+    'declined' => 'fa-hand-paper',
+    'revoked' => 'fa-eraser',
+    'superseded' => 'fa-plus'
+  }.freeze
+
   AVAILABLE_TYPES = ['all', 'submit', 'delete', 'add_role', 'change_devel', 'maintenance_incident', 'maintenance_release'].freeze
   AVAILABLE_STATES = ['new or review', 'new', 'review', 'accepted', 'declined', 'revoked', 'superseded'].freeze
 
   def request_state_color(state)
     STATE_COLORS[state.to_s] || ''
   end
 
+  def request_bootstrap_color(state)
+    STATE_BOOTSTRAP_COLORS[state.to_s] || ''
```

Comment:
Let's not cover for states we do not expect.

Reply:
why is the `to_s` needed? :thinking: would it make any different if this return `nil`?

Reply:
I guess sometimes this can be a symbol. I've just copied it from the method above...